core-bundle/src/EventListener/MakeBackendResponseUncacheableListener.php

```diff
@@ -29,6 +32,17 @@ public function __invoke(ResponseEvent $event): void
             return;
         }
 
+        $request = $event->getRequest();
+
+        if (
+            $request->headers->has('x-turbo-request-id')
+            && \in_array($request->getMethod(), [Request::METHOD_GET, Request::METHOD_HEAD], true)
```

Comment:
```suggestion
            && $request->isMethodCacheable()
```

like I already suggested. Why not take this helper?

Reply:
> like I already suggested. Why not take this helper?

<img width="657" height="130" alt="image" src="https://github.com/user-attachments/assets/596cb34d-9a61-4329-a438-2af7dcd80dc6" />

I did not see that anymore ;)

Reply:
See 2558b1d28d81ea2ddbba63fd8bafff75f0b0f46b